package/MDAnalysis/analysis/density.py

```diff
@@ -448,8 +426,20 @@ def density_from_Universe(universe, delta=1.0, atomselection='name OH2',
       soluteselection
             MDAnalysis selection for the solute, e.g. "protein" [``None``]
       cutoff
-            With *cutoff*, select '<atomsel> NOT WITHIN <cutoff> OF <soluteselection>'
-            (Special routines that are faster than the standard AROUND selection) [0]
+            With *cutoff*, select "<atomsel> NOT WITHIN <cutoff> OF <soluteselection>"
+            (Special routines that are faster than the standard ``AROUND`` selection)
+            [0]
+      update_selection
+            Should the selection of atoms be updated for every step? [``False``]
+            - ``True``: atom selection is updated for each frame, can be slow
+            - ``False``: atoms are only selected at the beginning
+      quiet
```

Comment:
When you already touch the doc string you can change it to the numpy-style format.